pxr/imaging/hdx/linearExposureScaleTask.h

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+#ifndef HDX_LINEAREXPOSURESCALE_TASK_H
+#define HDX_LINEAREXPOSURESCALE_TASK_H
+
+#include "pxr/pxr.h"
+#include "pxr/usd/sdf/path.h"
+#include "pxr/imaging/hdx/api.h"
+#include "pxr/imaging/hdx/task.h"
+#include "pxr/imaging/hdx/tokens.h"
+#include "pxr/imaging/hgi/graphicsCmds.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+/// \class HdxLinearExposureScaleTask
+///
+/// A task for applying an exposure scale for display.
+///
+class HdxLinearExposureScaleTask : public HdxTask
```

Comment:
I think this would read better as HdxExposureCompensationTask, as the verb rather than the noun where we're getting the exposure value from. (Likewise throughout the change).

Reply:
Makes sense to me!